src/Purpose.html

```diff
@@ -5725,44 +5720,72 @@
  
 &lt;choices&gt;[[[Hit the ground]|C1P61R1]] &lt;/choices&gt;</tw-passagedata><tw-passagedata pid="287" name="C1P61R1" tags="redirect" position="3075,15800" size="100,100">&lt;% window.story.redirect(&quot;C1P62&quot;, 5) %&gt;</tw-passagedata><tw-passagedata pid="288" name="C1P62" tags="page save" position="3075,15950" size="100,100">&lt;i&gt;&quot;Sarah? Sarah!&quot;&lt;/i&gt;, a voice yells, somewhere far away. I groan and start to stir.
 
-I&#39;m laying on my side, and there&#39;s someone near my face, again... Hopefully it&#39;s %Tiffany% here to offer me breakfast again... that would be nice...
+&lt;/div&gt;</tw-passagedata><tw-passagedata pid="287" name="C1P61R1" tags="redirect" position="3075,15800" size="100,100">&lt;% window.story.redirect(&quot;C1P62&quot;, 5) %&gt;</tw-passagedata><tw-passagedata pid="288" name="C1P62" tags="page save chase" position="3075,15950" size="100,100">&lt;flashback/&gt;
+&lt;ui&gt;standard&lt;/ui&gt;
+&lt;character.one.pain.neutral.neutral-backpack&gt;Sarah&lt;/character&gt;
+
+&lt;speech.unknown&gt;&quot;Sarah? Sarah!&quot;&lt;/speech&gt;
 
-I squint, my vision is blurry. There&#39;s a face, but... it has no eyes?
+&lt;action&gt;I groan and start to stir.&lt;/action&gt;
+&lt;action&gt;I&#39;m laying on my side, and there&#39;s someone near my face, again... Hopefully it&#39;s %Tiffany% here to offer me breakfast again... that would be nice...&lt;/action&gt;
```

Comment:
These could be split into several `action`s. As a general rule of thumb, anywhere there's a period (`.`), ellipses (`...`), or dashes (`-`) I split it into separate `action`s or `speech`s.